test/test_web_util.py

```diff
@@ -35,15 +35,14 @@ def test_json_response_tornado(self):
         """Tests JSON response output for Tornado"""
         mock_handler = MagicMock(spec=tornado.web.RequestHandler)
         test_data = {"key_1": "value", "key_2": 2}
-        expected_output = json.dumps({
+        expected_output = {
                 "code": 200,
                 "status": "Success",
                 "results": test_data,
-            }).encode("utf-8")
+            }
         res = web_util.echo_json_response(mock_handler, 200, "Success", test_data)
         self.assertTrue(res)
         mock_handler.set_status.assert_called_once_with(200)
-        mock_handler.set_header.assert_called_once_with('Content-Type', 'application/json')
```

Comment:
We're taking out the check of the mock for the method call, but is there another way we can test that the `Content-Type` header is set correctly?

Reply:
Not really because this is set by the handler which we are mocking. (I can also mock that, but this defeats the purpose of the test)